event/register_oracle_event.go

```diff
@@ -102,12 +103,16 @@ func verifyReportAndGetVoteOption(oracleRegistration *types.OracleRegistration,
 
 // makeOracleRegistrationVote makes a vote for oracle registration with VOTE_OPTION
 func makeOracleRegistrationVote(uniqueID, voterAddr, votingTargetAddr string, voteOption types.VoteOption, oraclePrivKey []byte, nodePubKey []byte) (*types.MsgVoteOracleRegistration, error) {
+	privKey, _ := crypto.PrivKeyFromBytes(oraclePrivKey)
+
 	pubKey, err := btcec.ParsePubKey(nodePubKey, btcec.S256())
 	if err != nil {
 		return nil, err
 	}
 
-	encryptedOraclePrivKey, err := crypto.Encrypt(pubKey, oraclePrivKey)
+	shareKey := crypto.ShareKey(privKey, pubKey)
+	// TODO: Nonce will be added.
+	encryptedOraclePrivKey, err := crypto.EncryptWithAES256(shareKey, nil, oraclePrivKey)
```

Comment:
The nonce may be in the `oracleRegistration` obtained using query client in line 57. I think you can add it now :)